shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -285,7 +301,10 @@ def find(self, entity_type, filters, fields=None, order=None, filter_operator=No
             )
         ]
 
-        # handle the ordering of the recordset
+        # Request additional fields from results
```

Comment:
I find the wording a bit confusing here, especially with `additional`. `Extract fields from rows` seems reasonable to me. Sorry to be nitpicky, but that comment feels confusing to me.